docs/HOW_TO_UPGRADE.md

```diff
@@ -0,0 +1,148 @@
+# HOW TO UPGRADE PARTICIPA.GENCAT.CAT
+
+## Steps to follow:
+In order to update the application correctly, the points commented in this file must be followed:
+
+Decidim are releasing new versions continuously. In order to get the latest one, update the dependencies:
+
+We recommend upgrading version to version to prevent errors.
+
+### Steps
+
+1. Go to the Gemfile and upgrade the decidim version to the next one stable.
+
+2. Go to the console and run
+```console
+bundle update decidim
+```
+
+3. Make sure you get all the latest migrations:
+```console
+bin/rails decidim:upgrade
```

Comment:
Since 0.15 or 0.16 migrations in non official modules are not copied anymore. It will be great to have a list here with all the customization modules that contain migrations.